grakn-graql/src/test/java/ai/grakn/graql/internal/query/match/MatchModifierTest.java

```diff
@@ -70,55 +70,38 @@ public void setUp() {
 
     @Test
     public void testOffsetQuery() {
-        MatchQuery query = qb.match(x.isa("movie").has("name", n)).orderBy(n, desc).offset(4);
+        GetQuery query = qb.match(x.isa("movie").has("name", n)).orderBy(n, desc).offset(4).get();
 
         assertResultsOrderedByValue(query, n, false);
     }
 
     @Test
     public void testLimitQuery() {
         Var t = var("t");
-        MatchQuery query = qb.match(x.isa("movie").has("title", t)).orderBy(t, asc).offset(1).limit(3);
+        GetQuery query = qb.match(x.isa("movie").has("title", t)).orderBy(t, asc).offset(1).limit(3).get();
 
         assertResultsOrderedByValue(query, t, true);
         assertEquals(3, query.stream().count());
     }
 
     @Test
     public void testOrPatternOrderByResource() {
-        MatchQuery query = qb.match(
+        GetQuery query = qb.match(
                 x.isa("movie").has("tmdb-vote-count", var("v")),
                 var().rel(x).rel(y),
                 or(
                         y.isa("person").has("name", "Marlon Brando"),
                         y.isa("genre").has("name", "crime")
                 )
-        ).orderBy("v", desc);
+        ).orderBy("v", desc).get();
 
         assertThat(query, variable(x, contains(MovieMatchers.godfather, MovieMatchers.godfather, MovieMatchers.apocalypseNow)));
     }
 
     @Test
-    public void testOrPatternOrderByUnselected() {
```

Comment:
Why is this not a relevant test anymore?

Reply:
Good point. I think I got confused about what it did. I'll try and add it back.